scania-digital-design-system/sdds
# fix(modal): standarise controls for modal

**Describe pull-request**  
Standardised controls and exposed js for modal stories.

**Solving issue**  
Fixes: [AB#2554](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2554)

**How to test**  
_Add description how to test if possible_
1. Go to storybook link below.
2. Check in Components -> Modal
3. Test the controls and check the preview code for both native and webcomponent story.

**Screenshots**  
-

**Additional context**  
-
